modules/lazyload/module.php

```diff
@@ -73,14 +73,11 @@ private function update_element_attributes( Element_Base $element ) {
 		}
 	}
 
-	private function reduce_background_image_size( $value, $css_property, $matches, $control ) {
+	private function remove_background_image( $value, $css_property, $matches, $control ) {
 		$is_lazyload_active = Utils::get_array_value_by_keys( $control, [ 'background_lazyload', 'active' ] );
 		if ( $is_lazyload_active ) {
 			if ( 0 === strpos( $css_property, 'background-image' ) && '{{URL}}' === $matches[0] ) {
-				$thumbnail_image_src = wp_get_attachment_image_src( $value['id'], 'thumbnail' );
-				if ( $thumbnail_image_src ) {
-					$value['url'] = $thumbnail_image_src[0];
-				}
+				$value['url'] = 'unset';
```

Comment:
What does it mean? why setting `unset`?

Reply:
To remove the thumbnails, I need to unset the background image.

Reply:
maybe use:
`$value['url'] = ''`
or
`unset($value['url'])`
?

Reply:
The `$value['url']` contains the CSS property `background-image` value. (unset), this is CSS not PHP.

Reply:
Update : i replace this with `none`